### PR TITLE
Improve docs of StabChain

### PR DESCRIPTION
**Pull request by ssiccha**
Use the proper option instead of a `SetSize` hack.

### CODE REVIEW

File: doc/ref/grpperm.xml [hunk 1]
```diff
@@ -360,8 +360,7 @@ things become faster and the result is guaranteed to be correct.
 <P/>
 <Log><![CDATA[
 gap> h:=Group( GeneratorsOfGroup( op ) );;
-gap> SetSize( h, 2317591180800 );
-gap> StabChain( h );;  time;
+gap> StabChain( h, rec( limit := 2317591180800 ) );;  time;
 170
```
Comment (fingolfin) [inline doc/ref/grpperm.xml:364]: Perhaps we should take a bigger example; for me the timings are not 1410 and 170 but rather 15 and 5 😂

**Review by fingolfin — APPROVED** [review]
Looks good to me

**Comment by hulpke** [comment]
I don't have an objection, but I would not consider `SetSize` a hack. In particular when walling `StabChain` only through other functions (say `SylowSubgroup`), it is the only way to provide size information.

**Review by hulpke — COMMENTED** [review]
Why do you use `limit` (which is an upper bound) and not `size` ?

**Comment by ssiccha** [comment]
I may have been a bit overzealous calling using `SetSize` a hack, but it is _not documented_ in `StabChain` that `Size`, if known, will be used during the computation. That probably should just be added to the docs of `StabChain`.

I use `limit` because it is a little more general. I guess `size` fits better here. In either case, a reference to the documentation of the options would make sense here.

**Comment by fingolfin** [comment]
So, how to proceed here? Merge as is? Change to `size := ...`? Add some text which references the options, and says something roughly like this:

> In the example below we used the `limit` option; here we could also have used the `size` option. Also `SetSize(G, ...)` prior to the StabChain call would have worked

**Review by wilfwilson — COMMENTED** [review]
Thanks @ssiccha for working to improve this.

I agree with the essence of what @hulpke is saying. I think `SetSize` was fine, but in any case, I think the `size` option is more appropriate here than `limit`.

If you want to go further, I suggest doing something like changing the paragraph preceding this example to following:
```
If you give the algorithm the order of the group, then it can check
its result, and so things become faster and the result is guaranteed
to be correct. This can be done with the <C>size</C> option
(see&nbsp;<Ref Func="StabChain" Label="for a group (and a record)"/>),
or by setting the size of the group beforehand with <C>SetSize</C>.
```
I don't think it's useful to mention the `limit` option here.

I second @fingolfin's suggestion of making the example more difficult, but that doesn't need to be done in this PR.

**Review by wilfwilson — APPROVED** [review]
I'm happy with what the new sentences say, although I'm biased since I basically wrote it 😉

**Pull request merged**